src/TodoListItem.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
+import { FaTrashAlt } from 'react-icons/fa';
+import style from './TodoListItem.module.css';
 
 function TodoListItem({ todo, onRemoveTodo }) {
+    const [isChecked, setIsChecked] = React.useState(false);
     function handleRemove() {
         onRemoveTodo(todo.id)
     }
-    return (<li >{todo.title}  <button type='button' onClick={handleRemove}>Remove</button>
+    function handleStrike() {
```

Comment:
Nicely done. Great naming convention as well. If you want to clean things up a bit, consider this, the logic is light and straightforward; directly apply the logic that manipulates `isChecked` to your `input` element.  Give it a try.